File: songs/elixir/03-songs.exs
defmodule Song do
  defstruct title: "", tags: []
end

defmodule Run do
  songs = [
    %Song{ title: "Cantaloupe Island",
           tags:  ["60s", "jazz"] },
    %Song{ title: "Let It Be",
           tags:  ["60s", "rock"] },
    %Song{ title: "Knockin' on Heaven's Door",
           tags:  ["70s", "rock"] },
    %Song{ title: "Emotion",
           tags:  ["70s", "pop"] },
    %Song{ title: "The River"}
  ]

  IO.inspect songs
end

File: songs/go/03-songs.go
package main

import "fmt"

type Song struct {
	Title string
	Tags  []string
}

func main() {
	songs := []Song{
		Song{"Cantaloupe Island",
			[]string{"60s", "jazz"}},
		Song{"Let It Be",
			[]string{"60s", "rock"}},
		Song{"Knockin' on Heaven's Door",
			[]string{"70s", "rock"}},
		Song{"Emotion",
			[]string{"70s", "pop"}},
		Song{"The River",
			nil},
	}

	for _, song := range songs{
		fmt.Println(song)
	}
}
